### ISSUE TITLE
Incidents not showing

### ISSUE THREAD

**Issue by SamKr** [coderrapp/codeRR.Server#61]
The server's up and running and incidents are being reported. They show up in the DB so communication is OK. 

However, in the webclient, the stats are shown but no actual incident information:

![image](https://user-images.githubusercontent.com/1439763/37455495-7cc9dba2-283d-11e8-9f91-e06cbf4cccda.png)

Tried multiple browsers. If you need login info for the dashboard I can provide them.

**Comment by jgauffin** [comment]
Sorry for that. Was a bit too fast yesterday. 

I'm on a winter vacation this week (downhill skiing), so I blame that.

Release v1.1.4 contains a fix.

![image from here](http://resources.mynewsdesk.com/image/upload/c_limit,dpr_1.25,f_auto,h_700,q_auto,w_690/d1opms7juwqmmof5wdpf.jpg)

**Comment by SamKr** [comment]
No problem =) Thanks for another quick assist. Great photo!

Do you have a paypal account so I can donate a whisky/beer for your time?

Related followup question; whenever I go to a specific application and look at the incidents, I see all incidents from all application (just like the main dashboard). Is that intended behaviour or am I missing a filtering setting or something?

**Comment by jgauffin** [comment]
I'm just glad that you filed an issue. No need to donate. 

That issue is related. We've changed the FindIncidents query a bit and had not regenerated the Typescript classes after the change. Hence the client side scripts did an incident search on the first page and the application page with outdated properties.

I've updated the zip files in the v1.1.4 release. Would you mind download it again and see if it works better.

**Comment by SamKr** [comment]
Grand, that fixed it. 

Sorry but yet another followup question, when I zoom in on a report, I get no context info:

![image](https://user-images.githubusercontent.com/1439763/37467293-1b3cccc8-2860-11e8-9707-c9f508827f08.png)

However I'm seeing more info in the DB. Perhaps related as well?

**Comment by jgauffin** [comment]
Yep, it's part of the same changes I was working on before going here. I've made the context information searchable (will be part of the v2.0 release that we are working on). It can currently take up to one minute before context information is attached to the error report. Will be faster later on.

However, it was not processed properly in the background job, so I've corrected that. Updated the v1.1.4 release again.

Guess it's time to use source code branches better.
